cmd/tidb-dashboard/cmd/kvmode.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/pingcap-incubator/tidb-dashboard/pkg/pd"
+	globalUtil "github.com/pingcap-incubator/tidb-dashboard/pkg/utils"
+)
+
+//var cfg = &DashboardCLIConfig{}
+
+var kvModeCmd = &cobra.Command{
+	Use:   "kvmode",
+	Short: "tikv mode related ops, including reset, clear tikv mode username, password",
+	Long:  ``,
+	Run: func(cmd *cobra.Command, args []string) {
+		if !kvModeDoAuth {
```

Comment:
I think this command does not need any checks as a parent command. You can move the auth check to the reset command.